### DISCUSSION TITLE
Property value group in Exchange item and property value in Function not returned

### DISCUSSION THREAD

**Discussion by khammoutene** [dbinfrago/py-capellambse#188] [Q&A] [answered]
Hello,

When I "print":

- an Exchange item, its property value groups are not "printed"
- a Function, its property values are not "printed"

How can I access them?

Is it an issue that is wider than these two Capella elements: Exchange item, Function?

Would be great to provide a work-around if it is not possible to get them via the simplified API.

Thank you

**Comment by ewuerger** [comment, marked as answer]
Hi,

from what I know is that we are working for a long time on updating the implementation for the PVMT extension. There is also an open PR that will come with a notebook giving a quick overview, capabilities and best practices regarding PVMT. For what I can give you now is the draft notebook [[Draft]08 Introduction to PVMT](https://github.com/DSD-DBS/py-capellambse/blob/pvmt-intro/examples/08%20Introduction%20to%20PVMT.ipynb) where the core functionality is shown and should be stable.

Hope that helps.
